src/ecrc-frontend/src/components/page/consent/Consent.js

```diff
@@ -222,17 +222,9 @@ export default function Consent({
       eivPassDetailsResults: "eivPassDetailsResults"
     };
 
-    let REACT_APP_FRONTEND_BASE_URL;
-
-    if (window.REACT_APP_FRONTEND_BASE_URL) {
-      REACT_APP_FRONTEND_BASE_URL = window.REACT_APP_FRONTEND_BASE_URL;
-    } else if (process.env.REACT_APP_FRONTEND_BASE_URL) {
-      REACT_APP_FRONTEND_BASE_URL = process.env.REACT_APP_FRONTEND_BASE_URL;
-    }
-
```

Comment:
this code being removed will require some code in `consent.test.js` to be removed as well.
specifically:
https://github.com/bcgov/jag-ecrc/blob/master/src/ecrc-frontend/src/components/page/consent/Consent.test.js#L200-L201

https://github.com/bcgov/jag-ecrc/blob/master/src/ecrc-frontend/src/components/page/consent/Consent.test.js#L226-L227

https://github.com/bcgov/jag-ecrc/blob/master/src/ecrc-frontend/src/components/page/consent/Consent.test.js#L261-L262

and 

https://github.com/bcgov/jag-ecrc/blob/master/src/ecrc-frontend/src/components/page/consent/Consent.test.js#L293

Reply:
Removed